### ISSUE TITLE
Allow building in Builder

### ISSUE THREAD

**Issue by bertob** [GeopJr/Hashbrown#28]
I just wanted to try and build the app locally in Builder, but looks like there's no Flatpak manifest in the repo. It'd be great to have that to make drive-by contributions and QA easier.

**Comment by GeopJr** [comment]
Would dropping the following manifest in `./extra/` work? From my attempts, I couldn't get Builder to not use the Makefile.

<details><summary

`extra/dev.geopjr.Hashbrown.yaml`

</summary>

```yaml
app-id: dev.geopjr.Hashbrown
runtime: org.gnome.Platform
runtime-version: '41'
sdk: org.gnome.Sdk
command: hashbrown
finish-args:
  - --socket=wayland
  - --socket=fallback-x11
  - --share=ipc
  - --device=dri

cleanup:
  - /include
  - /lib/pkgconfig
  - /share/doc
  - /share/man
  - '*.a'
  - '*.la'

modules:
  # Required by Crystal
  - name: livevent
    sources:
      - type: git
        url: https://github.com/libevent/libevent.git
        tag: release-2.1.12-stable
  # Required by Crystal for yaml parsing (/translations during compile-time)
  - name: libyaml
    sources:
      - type: archive
        url: https://github.com/yaml/libyaml/releases/download/0.2.5/yaml-0.2.5.tar.gz
        sha256: c642ae9b75fee120b2d96c712538bd2cf283228d2337df2cf2988e3c02678ef4
  # Required by libadwaita
  - name: libsass
    buildsystem: meson
    sources:
      - type: git
        url: https://github.com/lazka/libsass.git
        branch: meson
        commit: 302397c0c8ae2d7ab02f45ea461c2c3d768f248e
  # Required by libadwaita
  - name: sassc
    buildsystem: meson
    sources:
      - type: git
        url: https://github.com/lazka/sassc.git
        branch: meson
        commit: 82803377c33247265d779af034eceb5949e78354
  # Required by Hashbrown
  - name: libadwaita
    buildsystem: meson
    config-opts:
      - -Dexamples=false
      - -Dtests=false
    sources:
      - type: git
        url: https://gitlab.gnome.org/GNOME/libadwaita.git
        tag: 1.0.0.alpha.4
  # Hashbrown
  - name: hashbrown
    buildsystem: simple
    build-commands:
      - $(pwd)/crystal/bin/crystal build ./src/hashbrown.cr --no-debug --release
      - install -D -m 0755 hashbrown /app/bin/hashbrown
      - install -D -m 0644 extra/Hashbrown.desktop /app/share/applications/dev.geopjr.Hashbrown.desktop
      - install -D -m 0644 extra/icons/logo.svg /app/share/icons/hicolor/scalable/apps/dev.geopjr.Hashbrown.svg
      - install -D -m 0644 extra/icons/symbolic.svg /app/share/icons/hicolor/symbolic/apps/dev.geopjr.Hashbrown-symbolic.svg
    post-install:
      - install -D -m 0644 extra/dev.geopjr.Hashbrown.metainfo.xml /app/share/metainfo/dev.geopjr.Hashbrown.metainfo.xml
    sources:
      - type: dir
        path: ..
      - type: archive
        dest: crystal/
        url: https://github.com/crystal-lang/crystal/releases/download/1.2.1/crystal-1.2.1-1-linux-x86_64.tar.gz
        sha256: 66e51bd2e72363858b5a41755396158d6aaf76b177a51741e4ab2df3323f0ff4
      - type: git
        url: https://github.com/GeopJr/libadwaita.cr.git
        commit: 75bb4affac8350698eaba9fe39d6936e3cff8d7e
        dest: lib/libadwaita
      - type: git
        url: https://github.com/elorest/compiled_license.git
        tag: v1.0.0
        commit: ba052e31f8203c1db9997be3aba9dae62dc5f832
        dest: lib/compiled_license
```

</details>

**Comment by bertob** [comment]
IIRC the manifest needs to be json for it to work in Builder, so that might be the issue.

**Comment by GeopJr** [comment]
Worked! Thanks for raising this issue.

It should be fixed as of c10ca4dcdd991dbfb302a14a414dd82b0d6fca3d.
